tests/test_ribametamod.py

```diff
@@ -0,0 +1,322 @@
+import shutil
+from pathlib import Path
+
+import tomli_w
+
+from imod_coupler.__main__ import run_coupler
+
+
+def test_metamod(
+    tmp_path_dev: Path,
+    modflow_dll_devel: Path,
+    metaswap_dll_devel: Path,
+    metaswap_dll_dep_dir_devel: Path,
+    metaswap_lookup_table: Path,
+    bucket_ribametamod_loc: Path,
+) -> None:
+    """
+    Test if coupled ribametamod models run with the iMOD Coupler development version.
+    """
+    path_dev = tmp_path_dev / "bucket_model_driver_metamod"
+
+    shutil.copytree(bucket_ribametamod_loc, path_dev)
+
+    toml_path = path_dev / "imod_coupler.toml"
+    modflow6_dll = modflow_dll_devel
+    modflow6_model_dir = path_dev / "modflow6"
+    metaswap_dll = metaswap_dll_devel
+    metaswap_dll_dependency = metaswap_dll_dep_dir_devel
+    metaswap_model_dir = path_dev / "metaswap"
+
+    fill_para_sim_template(path_dev / "metaswap", metaswap_lookup_table)
+
+    write_metamod_toml(
+        path_dev,
+        modflow6_dll,
+        modflow6_model_dir,
+        metaswap_dll,
+        metaswap_dll_dependency,
+        metaswap_model_dir,
+    )
+    run_coupler(toml_path)
```

Comment:
`run_coupler` is mostly useful for debugging. Let's run it via `subprocess.run` like in the other tests to get process isolation

Reply:
done